SYCL/ESIMD/accessor_load_store.cpp

```diff
@@ -0,0 +1,116 @@
+//==------- accessor_gather_scatter.cpp  - DPC++ ESIMD on-device test ------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// TODO enable on Windows
+// REQUIRES: linux && gpu && opencl
+// RUN: %clangxx-esimd -fsycl %s -o %t.out
+// RUN: %ESIMD_RUN_PLACEHOLDER %t.out
```

Comment:
TODO to enable host test? Is there an issue filed for adding emulation support?

Reply:
We don't plan to support ESIMD on host device, only CPU emulation device (which appears as GPU device to SYCL RT). 

> Is there an issue filed for adding emulation support?

Do you mean for this particular test? Or for the feature in general?

Reply:
I mean emulation support for the feature in general.

Reply:
Not sure why this issue is needed and how this is related to this PR?

Reply:
Not required for this PR. Just for reference.